cmd/minikube/cmd/kubectl.go

```diff
@@ -41,7 +41,7 @@ var kubectlCmd = &cobra.Command{
 	Long: `Run the kubernetes client, download it if necessary.
 Examples:
```

Comment:
Can you explicitly call out and mention that `--` is required before passing any flags into kubectl?

Reply:
I agree ! I wonder is there a way we can add the -- ourselves for all of them ?

Reply:
This goes for all commands, including ssh. So it's nothing special for the "kubectl" command.

Would be nice if something could be done on the generic level, since it is the generated help...

Reply:
Added explicit warning